person.rb

```diff
@@ -18,4 +21,15 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission ? true : false
   end
+
+  def correct_name
+    @name
+  end
 end
+# #Check Using this code
+# person = Person.new(22, 'maximilianus')
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/208644038-c86c1560-1e03-4c27-8d32-66f059d70f63.png)


- [ ] You have a great readme file 👏 but this link is not working 😨 Kindly fix this link to make your readme file descriptive 😊 and to make your license file easily accessible for all the users 👍